src/graph/RebuildEdgeIndexExecutor.cpp

```diff
@@ -4,21 +4,21 @@
  * attached with Common Clause Condition 1.0, found in the LICENSES directory.
  */
 
-#include "graph/BuildEdgeIndexExecutor.h"
+#include "graph/RebuildEdgeIndexExecutor.h"
 
 namespace nebula {
 namespace graph {
 
-BuildEdgeIndexExecutor::BuildEdgeIndexExecutor(Sentence *sentence,
+RebuildEdgeIndexExecutor::RebuildEdgeIndexExecutor(Sentence *sentence,
                                                ExecutionContext *ectx) : Executor(ectx) {
-    sentence_ = static_cast<BuildEdgeIndexSentence*>(sentence);
+    sentence_ = static_cast<RebuildEdgeIndexSentence*>(sentence);
 }
 
-Status BuildEdgeIndexExecutor::prepare() {
+Status RebuildEdgeIndexExecutor::prepare() {
     return Status::OK();
 }
 
-void BuildEdgeIndexExecutor::execute() {
+void RebuildEdgeIndexExecutor::execute() {
```

Comment:
Where is the `isOffline_ ` check?  I think you'd better return an error when isOffline_ == false. right?

Reply:
auto isOffline = req.get_is_offline(); ?